flink-cdc-runtime/src/test/java/org/apache/flink/cdc/runtime/serializer/SerializerTestBase.java

```diff
@@ -265,8 +264,7 @@ void testSerializeIndividually() {
                 assertTrue("Trailing data available after deserialization.", in.available() == 0);
             }
         } catch (Exception e) {
-            System.err.println(e.getMessage());
```

Comment:
Could you check all system output? We should use LOGGER instead of System output.

Reply:
`System.err` has been purged, and all `System.out` usages have been confirmed appropriate.